mix.exs

```diff
@@ -30,8 +30,13 @@ defmodule VideoRoom.MixProject do
     [
       {:membrane_core,
        github: "membraneframework/membrane_core", branch: "use-push-flow-0.11", override: true},
-      {:membrane_rtc_engine, "~> 0.11.0", override: true},
+      {:membrane_rtc_engine, "~> 0.12.0", override: true},
       {:membrane_rtc_engine_timescaledb, "~> 0.1.0", runtime: false},
+      {
+        :membrane_rtp_plugin,
+        # path: "../membrane_rtp_plugin",
+        github: "membraneframework/membrane_rtp_plugin", branch: "master", override: true
```

Comment:
Obviously, we should wait with merge for the released versions

Reply:
It can be deleted altogether. Including only `membrane_rtc_engine` should be suffiecent since it depends on `rtp_plugin`. I've overlooked it and will change it immediately.